### ISSUE TITLE
characters employees

### ISSUE THREAD

**Issue by yoel123** [yoel123/java-text-based-space-station-manger-sim#5]
some employees are more important the other nameless expandable ones and contribute or hinder to the station's operation.

like the ship's doctor, accountant, constable, chief engineer etc etc.

some will have traits that add bonuses/ minuses to different stats (doesn't change stats permanently, just modifiers).

traits might also affect events.

their salary is higher than regular workers.

some will enable more game options like the accountant will enable the option to pay taxes automatically.

they also shod have an experience system as they get better at their work.

they will have their ow screen accessible from the station management screen.

the player can hire, fire, view stats, of characters.

**Comment by cShirley14** [comment]
This sounds along the lines of my thoughts from the story's theme. #9

**Comment by yoel123** [comment]
yes, the character class can be used for players too. letting the player chose his character starting trait/s.
